cmd/draft/draft.go

```diff
@@ -117,13 +128,42 @@ func teardown() {
 
 func ensureDraftClient(client *draft.Client) *draft.Client {
 	if client == nil {
-		return draft.NewClient(&draft.ClientConfig{
-			ServerAddr: draftHost,
-			Stdout:     os.Stdout,
-			Stderr:     os.Stderr,
-		})
+		return client
```

Comment:
The logic here means that if client == nil, return nil. That doesn’t seem right. :)

Reply:
oompf :1st_place_medal: